### PR TITLE
docs: update sample

### PR DESCRIPTION
**Pull request by tplooker**
## Description

Updates the sample in the repo to the latest release of this library

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] The commit message(s) follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Documentation has been added / updated (for bug fixes / features)
- [x] Changes follow the **[contributing](../docs/CONTRIBUTING.md)** document.

## Motivation and Context

See above

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Which merge strategy will you use?

- [x] Squash
- [ ] Rebase (REVIEW COMMITS)

### CODE REVIEW

File: sample/src/demo_multi.ts [hunk 1]
```diff
@@ -33,7 +32,6 @@ const documents: any = {
   "did:example:489398593#test": keyPairOptions,
   "did:example:489398593": exampleControllerDoc,
   "https://w3id.org/security/bbs/v1": bbsContext,
-  "https://w3id.org/security/v3-unstable": securityV3,
```
Comment (kdenhartog) [inline sample/src/demo_multi.ts:36]: Does removing this mean we no longer support v3-unstable context in our library and the bbs/v1 has to be used? I'm good with that, but let's make sure secureKey goes in the same direction for interop reasons.

Maybe we add this context as a normative statement to the LDP-BBS spec?

**Review by kdenhartog — APPROVED** [review]
Approving. This looks good. Myy question above may lead to additional considerations in other places, but won't affect this PR.

File: sample/src/demo_multi.ts (hunk 1 above)
Comment (tplooker) [reply sample/src/demo_multi.ts:36]: Na doesn't mean we dont support it just means its not required by the library to interact with the generated proofs

Comment (tplooker) [reply sample/src/demo_multi.ts:36]: We should discuss normative considerations around `@context` values in the [ldp-bbs](https://github.com/w3c-ccg/ldp-bbs2020) spec

**Pull request merged**